middleware/security_context_setter.rb

```diff
@@ -1,16 +1,29 @@
 module CloudFoundry
   module Middleware
     class SecurityContextSetter
+      PACKAGES_UPLOAD_REGEX = %r{/v3/packages/[\da-f]{8}-([\da-f]{4}-){3}[\da-f]{12}/upload}i.freeze
```

Comment:
What about a regex that is also valid for buildpacks (`/v3/buildpacks/.*/upload`) and droplets (`/v3/droplets/.*/upload`)? The form field is there for all upload endpoints and it would be more consistent.

Reply:
Yep fair enough. I've added the other endpoints as well.